gtsam/hybrid/HybridFactorGraph.cpp

```diff
@@ -79,6 +79,21 @@ void HybridFactorGraph::clear() {
   gaussianGraph_.resize(0);
 }
 
+DiscreteKeys HybridFactorGraph::discreteKeys() const {
+  DiscreteKeys result;
+  // Discrete keys from the discrete graph.
+  for (auto&& factor : discreteGraph_) {
```

Comment:
this should be a DiscreteGraph method, no?

Reply:
Yes!